### PR TITLE
Build releases using `GoReleaser`

### PR DESCRIPTION
**Pull request by jmatias**

### CODE REVIEW

**Comment by jmatias** [comment]
Fix `main.Version` reference in goreleaser.yml

File: .goreleaser-schema.json [hunk 1]
```diff
@@ -0,0 +1,2707 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"$ref": "#/definitions/Project",
+	"description": "goreleaser configuration definition file",
+	"definitions": {
```
Comment (svishwanath-tw) [inline .goreleaser-schema.json:5]: I wonder if this is static and specific to the `goreleaser` binary. 
Does this need to be checked in / maintained on `talisman` ?

**Review by svishwanath-tw — COMMENTED** [review]
Would be neat to see how `UPX` fits into this.

File: .goreleaser-schema.json (hunk 1 above)
Comment (jmatias) [reply .goreleaser-schema.json]: @svishwanath-tw 

Done. I pointed my IDE to https://raw.githubusercontent.com/goreleaser/goreleaser/v1.10.2/www/docs/static/schema.json. instead.

**Comment by svishwanath-tw** [comment]
@jmatias : At this point, the only thing preventing approval/merge is the increased binary size on non arm machines. Talisman build scripts used UPX to make the binary as small as can be.
My worry in going ahead is that a lot of github issues will be created asking to reduce the binary size again. Thoughts ?

**Comment by jmatias** [comment]
> @jmatias : At this point, the only thing preventing approval/merge is the increased binary size on non arm machines. Talisman build scripts used UPX to make the binary as small as can be.
> My worry in going ahead is that a lot of github issues will be created asking to reduce the binary size again. Thoughts ?

Regardless of GoReleaser, the issue would still be there for ARM (though they are working on it, we could try building the source code from their main branch)

That being said, 8 MB nowadays does not feel like a whole lot — I would argue that it's acceptable for now until we find a way to either tap into the build hooks or do it post release. (And by that I mean just doing an empty release and then writing something ourselves that uploads the binaries).

 I vote for keeping the 8 MB binaries for now.

**Review by svishwanath-tw — APPROVED** [review]

**Pull request merged**